### PR TITLE
Allow linking to version of documentation page

### PR DESCRIPTION
**Pull request by stbenjam**
Makes http://theforeman.org/documentation.html#1.9 work.

### CODE REVIEW

File: static/js/version-picker.js [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
   $('#manual').html('manual ' + version);
   $('#manual-link').attr('href', 'manuals/' + version + '/index.html');
   $('#release-notes').html('release notes ' + version);
   $('#release-notes-link').attr('href', 'manuals/' + version + '/index.html#Releasenotesfor' + version);
+}
+$(document).ready(function() {
+  var version = window.location.hash.substring(1);
+
+  if (version) {
+      setVersion(version);
```
Comment (domcleal) [inline static/js/version-picker.js]: Two space indentation, please

**Comment by domcleal** [comment]
This seems to permit XSS attacks, try accessing `/documentation.html#1.9<script>alert('hi')</script>`.  I think you'll need to validate or escape the version number.

File: static/js/version-picker.js [hunk 2]
```diff
@@ -1,8 +1,21 @@
+function setVersion(version) {
+  if (/^\d+\.\d+$/.test(version)) {
```
Comment (stbenjam) [inline static/js/version-picker.js:2]: Oops, good catch. How about this?

**Comment by domcleal** [comment]
Thanks @stbenjam!

**Pull request merged**